learn/react-native/disable-device-screencapture.md

```diff
@@ -0,0 +1,98 @@
+unauthorized users are prevented from capturing and potentially misusing this sensitive information. Secondly, it safeguards 
+intellectual property by preventing users from easily capturing and distributing proprietary or copyrighted content without permission. 
+This is particularly relevant for apps that offer paid content, as it helps prevent content piracy. Additionally, 
+disabling screenshots ensures user privacy by preventing accidental or unintentional sharing of sensitive content, conversations, or activities. 
+It also helps app developers comply with industry regulations regarding data security and privacy, especially in fields 
+like finance or healthcare. By implementing measures to protect user data and privacy, such as disabling screenshots, 
+app developers can enhance user trust and confidence. It's important to note that disabling screenshots alone is not 
+foolproof, so it's crucial to implement other security measures like encryption and secure data handling.
+
+## Expo's ScreenCapture Plugin
+
+[ScreenCapture](https://docs.expo.dev/versions/latest/sdk/screen-capture/) plugin empowers developers by offering granular control over screen capture functionality in their 
+apps. By enabling or disabling screenshots programmatically, developers can enhance security and privacy. 
+Preventing users from capturing sensitive content, such as confidential data or premium resources, safeguards user 
+information and protects intellectual property. With a seamless integration within the Expo ecosystem, the ScreenCapture
+plugin ensures a smooth implementation process. Developers can easily enforce screenshot restrictions, giving them greater 
+control over app security and privacy. By leveraging Expo's ScreenCapture plugin, developers can create more secure and 
+privacy-conscious apps, delivering a safer and more confidential user experience.
+
+## Adding ScreenCapture to your WaveMaker React Native App
+
+ScreenCapture plugin can be installed in few steps in WaveMaker. Please refer to this [page](https://docs.wavemaker.com/learn/react-native/third-party-expo-plugins#expo)
+on how to install a plugin.
+
+## Using ScreenCapture in your WaveMaker App
+
+ScreenCapture plugin can be leveraged in your WaveMaker app through the following ways:
+
+- Enabling and Disabling ScreenCapture on Demand
+- Disabling ScreenCapture on App Launch
+
+### Enabling and Disabling ScreenCapture on Demand
+
+Some apps may require the ability to enable or disable screen capture only in certain situations i.e. on demand.
+For example, a video streaming app may want to allow screenshots when the user is watching a free video, but disable for
+content that requires a paid subscription.
+
+For our example, we will create a simple app that allows users to enable or disable screenshots on click of a button. 
+The following is the mockup of the app:
+
+![ScreenCaptureOnDemand](/learn/assets/ScreeCaptureOnDemand.png)
```

Comment:
the image size should be adjusted to a typical mobile size

Reply:
Two options: 

1. Please try resizing the image to reduce its size by around 35%. You can use the paint or image editing option from your machine and then replace the image. 
2. `import pluginExample from '/learn/assets/imagename.png';`   // place this at the start of the doc before the first paragraph

`<img src={pluginExample} style={{width:"35%"}} />`   // place at the end of the document.

Reply:
Updated